lib/chef/util/dsc/local_configuration_manager.rb

```diff
@@ -78,6 +78,18 @@ def lcm_command_code(configuration_path, test_only_parameters)
 EOH
     end
 
+    def test_lcm_command_code
+      ps_version = Mixlib::ShellOut.new("powershell $PSVersionTable.PSVersion.Major").run_command
+      ps_version.error!
+      if ps_version.stdout.to_i > 4
+        <<-EOH
+$ProgressPreference = 'SilentlyContinue';Test-DscConfiguration -path #{@configuration_path}
```

Comment:
Does this mean we still run `Start-DscConfiguration` and then `Test-DscConfiguration` to test? Wouldn't `Start-DscConfiguration` without `-whatif` actually make a change? I think `test_only_parameters` need to something more like `test_command`.

Reply:
First we run `Test-DscConfiguration` then `Start-DscConfiguration`.